xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -148,9 +184,24 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns the current snapshot of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> state that must
+            be captured by the caller and passed back in to the <see cref="T:System.Text.Json.Utf8JsonWriter" /> constructor with more data.</summary>
+        <returns>The current snapshot of the <see cref="T:System.Text.Json.Utf8JsonWriter" /> state.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+Unlike the <xref:System.Text.Json.Utf8JsonWriter>, which is a ref struct, the state can survive across async/await boundaries, and hence this type is required to provide support for reading
```

Comment:
Not reading in this case, but writing. This issue appears in our source comments too:
https://github.com/dotnet/corefx/blob/ce62a086814fdfecea1d6ba6657c3fb88763e1d6/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs#L84